pkg/detectors/signable/signable.go

```diff
@@ -28,7 +28,7 @@ var (
 // Keywords are used for efficiently pre-filtering chunks.
 // Use identifiers in the secret preferably, or the provider name.
 func (s Scanner) Keywords() []string {
-	return []string{"signable"}
+	return []string{"\bsignable"}
```

Comment:
This is used for substring matching. You might be looking to modify the regex pattern in the file